maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -16495,6 +16480,7 @@
 /obj/structure/reagent_dispensers/fueltank,
 /turf/open/floor/plating/plating_catwalk,
 /area/almayer/maint/hull/upper/u_m_p)
+>>>>>>> master
```

Comment:
these are merge conflict markers that you've committed, and you'll need to fix your map

Reply:
It also indicates you do not have the merge hooks installed. I highly recommend you do so conflicts like this are easier to deal with.